project.go

```diff
@@ -0,0 +1,391 @@
+// Copyright 2016 EF CTX. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/tsuru/gnuflag"
+	"github.com/tsuru/tsuru/cmd"
+	tsuruerrors "github.com/tsuru/tsuru/errors"
+)
+
+type projectCreate struct {
+	fs       *gnuflag.FlagSet
+	name     string
+	platform string
+	team     string
+	plan     string
+	envs     commaSeparatedFlag
+}
+
+func (*projectCreate) Info() *cmd.Info {
+	return &cmd.Info{
+		Name: "project-create",
+		Desc: "creates a remote project in the tranor server",
+	}
+}
+
+func (c *projectCreate) Run(ctx *cmd.Context, client *cmd.Client) error {
+	if c.name == "" || c.platform == "" {
+		return errors.New("please provide the name and the platform")
+	}
+	config, err := loadConfigFile()
+	if err != nil {
+		return errors.New("unable to load environments file, please make sure that tranor is properly configured")
+	}
+	err = c.envs.validate(config.envNames())
+	if err != nil {
+		return fmt.Errorf("failed to load environments: %s", err)
+	}
+	envs := c.getEnvironmentsByName(config.Environments, c.envs.Values())
+	apps, err := c.createApps(envs, client)
+	if err != nil {
+		return err
+	}
+	err = c.setCNames(apps, client)
+	if err != nil {
+		appNames := make([]string, len(apps))
+		for i, app := range apps {
+			appNames[i] = app["name"]
+		}
+		deleteApps(appNames, client)
+		return fmt.Errorf("failed to configure project %q: %s", c.name, err)
+	}
+	fmt.Fprintf(ctx.Stdout, "successfully created the project %q!\n", c.name)
+	if gitRepo := apps[0]["repository_url"]; gitRepo != "" {
```

Comment:
Can't we have some issue by using the first repo from the slice `apps`? How are they sorted?

Ideally, we'd have nice-looking git repos URLs for projects, agnostic to environment:
eg:  `git@git.tsuru.example.com:<project-name>.git`

Which would not change in case we wanted to update a project, (ex: add other environments seems a valid use case).

However, it seems this may not be possible, as it may involve changing Tsuru. Could you confirm, @fsouza?